### PR TITLE
Bootloader mode wipe_device also erases firmware

### PR DESCRIPTION
**Pull request by solipsis**
Similar behavior to how Trezor handles wipe device in bootloader mode

### CODE REVIEW

File: tools/bootloader/usb_flash.c [hunk 1]
```diff
@@ -519,20 +520,28 @@ void handler_wipe(WipeDevice *msg)
...
+    /* Erase config data sectors  */
+    for (uint32_t i = FLASH_STORAGE1; i <= FLASH_STORAGE3; i++) {
+        bl_flash_erase_word(i);
+        layoutProgress("Erasing Secrets", (i * 1000 / 9));
+    }
+
+    /* Erase application section */
+    for (int i = 7; i <= 11; ++i) {
+        bl_flash_erase_sector(i);
+        layoutProgress("Erasing firmware", ((i + 3 - 7 + 2) * 1000 / 9));
```
Comment (keepkeyjon) [inline tools/bootloader/usb_flash.c]:
```suggestion
        layoutProgress("Erasing Firmware", ((i + 3 - 7 + 2) * 1000 / 9));
```

File: tools/bootloader/usb_flash.c [hunk 2]
```diff
@@ -519,20 +520,28 @@ void handler_wipe(WipeDevice *msg)
...
     }
 
+    flash_lock();
+    layoutProgress("Erasing firmware", 1000);
```
Comment (keepkeyjon) [inline tools/bootloader/usb_flash.c]:
```suggestion
    layoutProgress("Erasing Firmware", 1000);
```

File: tools/bootloader/usb_flash.c [hunk 3]
```diff
@@ -519,20 +520,28 @@ void handler_wipe(WipeDevice *msg)
...
 
-    // Only erase the active sector, leaving the other two alone.
-    Allocation storage_loc = FLASH_INVALID;
-    if(find_active_storage(&storage_loc) && storage_loc != FLASH_INVALID) {
-        bl_flash_erase_word(storage_loc);
-        flash_lock();
+    /* Erase config data sectors  */
+    for (uint32_t i = FLASH_STORAGE1; i <= FLASH_STORAGE3; i++) {
+        bl_flash_erase_word(i);
+        layoutProgress("Erasing Secrets", (i * 1000 / 9));
```
Comment (keepkeyjon) [inline tools/bootloader/usb_flash.c]:
```suggestion
        layoutProgress("Erasing Secrets", (i * 1000 / 8));
```

File: tools/bootloader/usb_flash.c [hunk 4]
```diff
@@ -583,8 +592,8 @@ void handler_erase(FirmwareErase *msg)
 
             /* Erase application section */
             for (int i = 7; i <= 11; ++i) {
-                bl_flash_erase_word(i);
                 layoutProgress("Preparing for upgrade", ((i + 3 - 7 + 2) * 1000 / 9));
+                bl_flash_erase_word(i);
```
Comment (keepkeyjon) [inline tools/bootloader/usb_flash.c]: don't worry about this one... I'll get it in my PR.

Comment (solipsis) [reply tools/bootloader/usb_flash.c]: fixed

File: tools/bootloader/usb_flash.c (hunk 3 above)
Comment (solipsis) [reply tools/bootloader/usb_flash.c]: fixed

File: tools/bootloader/usb_flash.c (hunk 2 above)
Comment (solipsis) [reply tools/bootloader/usb_flash.c]: fixed

**Pull request merged**